bc/web/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+from bc.core.models import AbstractDateTimeModel
+
+
+class BotSuggestion(AbstractDateTimeModel):
+    name = models.CharField(
+        help_text="Name of the user making the suggestion",
+        max_length=100,
+    )
+    platform = models.TextField(
+        help_text="List of platforms where the bot would work"
+    )
+    use_case = models.TextField(
+        help_text="Description of bot and the kinds of cases it would follow."
+    )
+
+
+class WaitList(AbstractDateTimeModel):
+    platform = models.TextField(
+        help_text="List of platforms to include features of the bots"
+    )
```

Comment:
I set up the choices in the form because it allows us to select multiple values (I think We should allow this behavior because We're using checkboxes to show the options) and then store the data in the db.

If We use the `choices` argument of a field, We can only store one of the values in the list, so If a user wants to get the features of the bot in two or more platforms but We set up the choices in the model class, He/She will be forced to select just one of the options.

Reply:
Oh, yeah, I see what you mean. I guess the only way to do it in the model is to do a related model and that's wildly more than we need. Carry on!